samples/dotnet/kernel-syntax-examples/Example23_ReadOnlyMemoryStore.cs

```diff
@@ -76,13 +76,15 @@ public Task<bool> DoesCollectionExistAsync(string collectionName, CancellationTo
             throw new System.NotImplementedException();
         }
 
-        public Task<MemoryRecord?> GetAsync(string collectionName, string key, CancellationToken cancel = default)
+        public Task<MemoryRecord?> GetAsync(string collectionName, string key, bool withEmbedding = default, CancellationToken cancel = default)
```

Comment:
[nit] `bool withEmbedding = false`

Reply:
@adrianwyatt ah I got feedback earlier suggesting that I set this to default as booleans default to false. Happy to change it back to be more clear.

Reply:
done